StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsDataAccessNotice.swift

```diff
@@ -8,11 +8,12 @@
 import Foundation
 
 struct FinancialConnectionsDataAccessNotice: Decodable {
+    let icon: FinancialConnectionsImage?
     let title: String
     let subtitle: String?
     let body: Body
     let connectedAccountNotice: String?
-    let learnMore: String
+    let disclaimer: String?
```

Comment:
[link to model info for double-checking](https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/text/data/data_access_notice_v3.rb#L5-15) (important they match backend)